Dockerfile

```diff
@@ -1,11 +1,33 @@
 FROM python:3.12-bookworm AS dev
 
+RUN apt-get update && \
+    apt-get install build-essential debhelper devscripts equivs dh-virtualenv python3-venv dh-sysuser dh-exec -y && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV PATH="/root/miniconda3/bin:${PATH}"
+RUN ARCH=$(uname -m) && \
+    if [ "$ARCH" = "x86_64" ]; then \
+        MINICONDA_URL="https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh"; \
+    elif [ "$ARCH" = "aarch64" ]; then \
```

Comment:
Same comment. Are we finally supporting `ARM`?

Reply:
Here yes, if you're using Apple Silicon we need the above to build compatible images.